.github/workflows/ci.yml

```diff
@@ -103,14 +116,26 @@ jobs:
         run: php php-cs-fixer fix --rules @PHP73Migration,@PHP71Migration:risky,blank_line_after_opening_tag -q
         if: "matrix.execute-migration-rules == 'yes'"
 
+      - name: Upgrade PHPUnit to calculate code coverage - this step won't be needed after merging https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/5314
+        if: matrix.calculate-code-coverage == 'yes'
+        run: |
+          composer require --dev phpunit/phpunit:^8 --with-all-dependencies
+          php php-cs-fixer fix --rules=void_return -q tests
+
       - name: Run tests
         continue-on-error: ${{ matrix.php-version == '8.0' }}
         env:
           PHP_CS_FIXER_IGNORE_ENV: ${{ matrix.PHP_CS_FIXER_IGNORE_ENV }}
           FAST_LINT_TEST_CASES: ${{ matrix.FAST_LINT_TEST_CASES }}
           PHP_CS_FIXER_TEST_USE_LEGACY_TOKENIZER: ${{ matrix.PHP_CS_FIXER_TEST_USE_LEGACY_TOKENIZER }}
           SYMFONY_DEPRECATIONS_HELPER: ${{ matrix.SYMFONY_DEPRECATIONS_HELPER }}
-        run: vendor/bin/phpunit
+        run: vendor/bin/phpunit ${{ matrix.phpunit-flags }}
+
+      - name: Upload coverage results to Coveralls
+        if: matrix.calculate-code-coverage == 'yes'
+        env:
+          COVERALLS_REPO_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: php vendor/bin/php-coveralls --verbose
```

Comment:
If this could follow the approach with `run: | <linebreak> code` other steps use, that'd great.

If you pass the command on the next line, and if you had to add another command to it, there will be one-line diff instead of three-liner. And it's easier to read: commands are separated from overall YAML.

Reply:
Good idea, will do it after https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/5314 is merged.

Reply:
> other steps use, that'd great.

Not anymore: https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/5330

My reasoning is let's not make it bigger (in terms of lines) than it has to be - if it's one command we should have it as string and array of string when there are more commands.